src/screens/CompareModels/CompareModels.js

```diff
@@ -59,17 +86,85 @@ export function CompareModels({ match, location }) {
     });
   }
 
+  const [sortType, setSortType] = useState(SORT_TYPES.ALPHABETICAL);
+  const [filterTypeIntervention, setFilterTypeIntervention] = useState(
+    DEFAULT_INTERVENTION_FILTER,
+  );
```

Comment:
FWIW we're down to 1 state that's on "limited action" and ~10 that are on "social distancing", so the usefulness of this filtering may decline. :-)

Reply:
I think I'll leave it because it future we may want to add more filter functionality (open to hiding it for now though). Thoughts?

Reply:
Sounds good to me!